p2p/src/net/libp2p/backend.rs

```diff
@@ -18,6 +18,8 @@
 
 // TODO: think about connection management
 
+// TODO: think about connection management
```

Comment:
?

Reply:
It's appear after rebase. I didn't add it